test/sqlsrv/connection_resiliency_keywords.phpt

```diff
@@ -0,0 +1,245 @@
+            \[message\] => \[Microsoft\]\[ODBC Driver 1[1-9] for SQL Server\]Invalid value specified for connection string attribute 'ConnectRetryCount'
+        \)
+
+\)
+Could not connect on fourth attempt.
+Array
+\(
+    \[0\] => Array
+        \(
+            \[0\] => 08001
+            \[SQLSTATE\] => 08001
+            \[1\] => 0
+            \[code\] => 0
+            \[2\] => \[Microsoft\]\[ODBC Driver 1[1-9] for SQL Server\]Invalid value specified for connection string attribute 'ConnectRetryInterval'
+            \[message\] => \[Microsoft\]\[ODBC Driver 1[1-9] for SQL Server\]Invalid value specified for connection string attribute 'ConnectRetryInterval'
+        \)
+
+\)
+Could not connect on fifth attempt.
+Array
+\(
+    \[0\] => Array
+        \(
+            \[0\] => 08001
+            \[SQLSTATE\] => 08001
+            \[1\] => 0
+            \[code\] => 0
+            \[2\] => \[Microsoft\]\[ODBC Driver 1[1-9] for SQL Server\]Invalid value specified for connection string attribute 'ConnectRetryCount'
+            \[message\] => \[Microsoft\]\[ODBC Driver 1[1-9] for SQL Server\]Invalid value specified for connection string attribute 'ConnectRetryCount'
+        \)
+
+\)
+Could not connect on sixth attempt.
+Array
+\(
+    \[0\] => Array
+        \(
+            \[0\] => IMSSP
+            \[SQLSTATE\] => IMSSP
+            \[1\] => -33
+            \[code\] => -33
+            \[2\] => Invalid value type for option ConnectRetryCount was specified.  Integer type was expected.
+            \[message\] => Invalid value type for option ConnectRetryCount was specified.  Integer type was expected.
+        \)
+
+\)
+Could not connect on seventh attempt.
+Array
+\(
+    \[0\] => Array
+        \(
+            \[0\] => IMSSP
+            \[SQLSTATE\] => IMSSP
+            \[1\] => -33
+            \[code\] => -33
+            \[2\] => Invalid value type for option ConnectRetryInterval was specified.  Integer type was expected.
+            \[message\] => Invalid value type for option ConnectRetryInterval was specified.  Integer type was expected.
+        \)
+
+\)
+Could not connect on eighth attempt.
+Array
+\(
+    \[0\] => Array
+        \(
+            \[0\] => IMSSP
+            \[SQLSTATE\] => IMSSP
+            \[1\] => -8
+            \[code\] => -8
+            \[2\] => An invalid connection option key type was received. Option key types must be strings.
+            \[message\] => An invalid connection option key type was received. Option key types must be strings.
+        \)
+
+\)
+Could not connect on ninth attempt.
+Array
+\(
+    \[0\] => Array
+        \(
+            \[0\] => IMSSP
+            \[SQLSTATE\] => IMSSP
+            \[1\] => -8
+            \[code\] => -8
+            \[2\] => An invalid connection option key type was received. Option key types must be strings.
+            \[message\] => An invalid connection option key type was received. Option key types must be strings.
+        \)
+
+\)
```

Comment:
These two tests are very long although they are just doing the same thing with different ConnectRetryCount and ConnectRetryInterval 9 times for each test. Probably looks cleaner if you can make a function and all it nine times with different params.